pages/Electric-Dust-Washer.md

```diff
@@ -0,0 +1,35 @@
+The Electric Dust Washer is a Machine in the [Electric Machines](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Electric-Machines) category. It obtains from [Dusts](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Dusts) from [Sifted Ore](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Sifted-Ore) without any by-products.
+
+## Obtaining
+
+The Electric Dust Washer is craftable using the [Enchanted Crafting Table](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Enhanced-Crafting-Table). The recipies vary with the machine levels and can be referred to in the Slimefun guide. The Electric Dust Washer has 3 levels of efficiency and power consumtion. The various Electric Dust Washers look like the following:
+
+![Electric Dust Washer](https://raw.githubusercontent.com/Slimefun/Slimefun-Wiki/master/images/electric-dust-washer-1.png)
+
+![Electric Dust Washer (II)](https://raw.githubusercontent.com/Slimefun/Slimefun-Wiki/master/images/electric-dust-washer-2.png)
+
+![Electric Dust Washer (III)](https://raw.githubusercontent.com/Slimefun/Slimefun-Wiki/master/images/electric-dust-washer-3.png)
+
+## Usage
+
+The Electric Dust Washer has to be connected to an [Electrical Network](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Electric-Machines) with sufficient energy to function.
+The slots of an Electric Dust Washer are as follows:
+![Electric Dust Washer Slots](https://raw.githubusercontent.com/Slimefun/Slimefun-Wiki/master/images/electric-dust-washer-slots.png)
+The slot on the left (highlited in cyan stained glass pane) are the input slots where the [Sifted Ore](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Sifted-Ore) is taken as input.
+The slot on the right (highlited in orange stained glass pane) are the output slots where the [Dusts](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Dusts) are produced. Once the output slots are filled the Electric Dust Washer Stops its operation untill the output slots are cleared. Therefore, a [Cargo Management System](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Cargo-Management) is the most efficient way to handle a Electric Dust Washer and can be used to create complex automated dust obtaining systems.
```

Comment:
Typo, change to: "highlighted by orange stained glass panes".

Old link again.

Change to "the output slots where the produced dusts are placed to".

Typo in "untill".

Decapitalise "S" in "Stops"